2025-09-01 19:35:52,158 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2025-09-01 19:35:52,160 INFO supervisord started with pid 7
2025-09-01 19:35:53,162 INFO spawned: 'app' with pid 8
2025-09-01 19:35:53,163 INFO spawned: 'nginx' with pid 9
2025-09-01 19:35:54,176 INFO success: app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2025-09-01 19:35:54,176 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
[CTRL] POST /setup
[Time: 01.09.25 19:36:06] [Level: info] id: 86cd0362d8befde0
[Time: 01.09.25 19:36:06] [Level: info] coordinator: False
[Time: 01.09.25 19:36:06] [Level: info] clients: ['86cd0362d8befde0', 'd7b8ee50641aecc8', '994a9588c6a490fd']
[Time: 01.09.25 19:36:06] [Level: info] state: initial
[Time: 01.09.25 19:36:06] [Level: info] [State: initial] [CLIENT] Starting FedBayNet...
[Time: 01.09.25 19:36:06] [Level: info] [State: initial] [CLIENT] Node ID: 86cd0362d8befde0, Coordinator: False
[Time: 01.09.25 19:36:06] [Level: info] [State: initial] Initial to Read Input
[Time: 01.09.25 19:36:06] [Level: info] transition: read input
[Time: 01.09.25 19:36:07] [Level: info] state: read input
[Time: 01.09.25 19:36:07] [Level: info] [State: read input] [CLIENT] Reading dataset and configuration...
[Time: 01.09.25 19:36:07] [Level: info] [State: read input] Read Input to Local Computation
[Time: 01.09.25 19:36:07] [Level: info] [State: read input] [CLIENT] Configuration Loaded Successfully
[Time: 01.09.25 19:36:07] [Level: info] [State: read input] [CLIENT] Loaded dataset from /mnt/input: 57 records and 28 variables
[Time: 01.09.25 19:36:07] [Level: info] [State: read input] [CLIENT] Error reading config: Expecting value: line 1 column 1 (char 0)
[Time: 01.09.25 19:36:07] [Level: info] transition: read input
Traceback (most recent call last):
  File "/app/states.py", line 74, in run
    expknowledge = json.load(f)
  File "/usr/local/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[Time: 01.09.25 19:36:08] [Level: info] state: read input
[Time: 01.09.25 19:36:08] [Level: info] [State: read input] [CLIENT] Reading dataset and configuration...
[Time: 01.09.25 19:36:08] [Level: info] [State: read input] Read Input to Local Computation
[Time: 01.09.25 19:36:08] [Level: info] [State: read input] [CLIENT] Configuration Loaded Successfully
[Time: 01.09.25 19:36:08] [Level: info] [State: read input] [CLIENT] Loaded dataset from /mnt/input: 57 records and 28 variables
[Time: 01.09.25 19:36:08] [Level: info] [State: read input] [CLIENT] Error reading config: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
[Time: 01.09.25 19:36:08] [Level: info] transition: read input
  File "/app/states.py", line 74, in run
    expknowledge = json.load(f)
  File "/usr/local/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[Time: 01.09.25 19:36:09] [Level: info] state: read input
[Time: 01.09.25 19:36:09] [Level: info] [State: read input] [CLIENT] Reading dataset and configuration...
[Time: 01.09.25 19:36:09] [Level: info] [State: read input] Read Input to Local Computation
[Time: 01.09.25 19:36:09] [Level: info] [State: read input] [CLIENT] Configuration Loaded Successfully
[Time: 01.09.25 19:36:09] [Level: info] [State: read input] [CLIENT] Loaded dataset from /mnt/input: 57 records and 28 variables
[Time: 01.09.25 19:36:09] [Level: info] [State: read input] [CLIENT] Error reading config: Expecting value: line 1 column 1 (char 0)
[Time: 01.09.25 19:36:09] [Level: info] transition: read input
Traceback (most recent call last):
  File "/app/states.py", line 74, in run
    expknowledge = json.load(f)
  File "/usr/local/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[Time: 01.09.25 19:36:10] [Level: info] state: read input
[Time: 01.09.25 19:36:10] [Level: info] [State: read input] [CLIENT] Reading dataset and configuration...
[Time: 01.09.25 19:36:10] [Level: info] [State: read input] Read Input to Local Computation
[Time: 01.09.25 19:36:10] [Level: info] [State: read input] [CLIENT] Configuration Loaded Successfully
[Time: 01.09.25 19:36:10] [Level: info] [State: read input] [CLIENT] Loaded dataset from /mnt/input: 57 records and 28 variables
Traceback (most recent call last):
[Time: 01.09.25 19:36:10] [Level: info] [State: read input] [CLIENT] Error reading config: Expecting value: line 1 column 1 (char 0)
  File "/app/states.py", line 74, in run
    expknowledge = json.load(f)
  File "/usr/local/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[Time: 01.09.25 19:36:10] [Level: info] transition: read input
[Time: 01.09.25 19:36:11] [Level: info] state: read input
[Time: 01.09.25 19:36:11] [Level: info] [State: read input] [CLIENT] Reading dataset and configuration...
[Time: 01.09.25 19:36:11] [Level: info] [State: read input] Read Input to Local Computation
[Time: 01.09.25 19:36:11] [Level: info] [State: read input] [CLIENT] Configuration Loaded Successfully
[Time: 01.09.25 19:36:11] [Level: info] [State: read input] [CLIENT] Loaded dataset from /mnt/input: 57 records and 28 variables
[Time: 01.09.25 19:36:11] [Level: info] [State: read input] [CLIENT] Error reading config: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/app/states.py", line 74, in run
    expknowledge = json.load(f)
  File "/usr/local/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[Time: 01.09.25 19:36:11] [Level: info] transition: read input
[Time: 01.09.25 19:36:12] [Level: info] state: read input
[Time: 01.09.25 19:36:12] [Level: info] [State: read input] [CLIENT] Reading dataset and configuration...
[Time: 01.09.25 19:36:12] [Level: info] [State: read input] Read Input to Local Computation
[Time: 01.09.25 19:36:12] [Level: info] [State: read input] [CLIENT] Configuration Loaded Successfully
[Time: 01.09.25 19:36:12] [Level: info] [State: read input] [CLIENT] Loaded dataset from /mnt/input: 57 records and 28 variables
[Time: 01.09.25 19:36:12] [Level: info] [State: read input] [CLIENT] Error reading config: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/app/states.py", line 74, in run
[Time: 01.09.25 19:36:12] [Level: info] transition: read input
    expknowledge = json.load(f)
  File "/usr/local/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
